msteams-platform/includes/tabs/content-security-policy-headers.md

```diff
@@ -1,5 +1,16 @@
-|Microsoft 365 app host| frame-ancestor permission|
+> [!WARNING]
+> Microsoft's cloud services, including web versions of Teams (*teams.microsoft.com*), Outlook (*outlook.com*), and Microsoft 365 (*microsoft365.com*) domains are migrating to the new *cloud.microsoft* domain by June 1, 2024. The migration affects your app's critical functionality and requires you to perform the following steps before June 1, 2024:
+>
+> 1. Update TeamsJS SDK to v.2.19.0 or higher. For more information about the latest release of TeamsJS SDK, see [@microsoft/teams-js - npm](https://www.npmjs.com/package/@microsoft/teams-js).
+>
+> 2. Update your [`X-Frame-Options`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options) or [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) headers in your Teams app to allow your app to access the *teams.cloud.microsoft* domain.
+>
+> 3. If your Teams app extends across Outlook and Microsoft 365, ensure you allow your app to access *teams.cloud.microsoft*, *outlook.cloud.microsoft*, and *m365.cloud.microsoft* domains.
```

Comment:
Could we further emphasize the new domains in **bold** (rather than just italics) to make it as easy as possible for customers to pick out the changes they need to add to their app CSP headers?

Reply:
Excellent suggestion! Done.